manifests/init.pp

```diff
@@ -1263,4 +1263,18 @@
       fail("Pam is only supported on RedHat, SuSE, Debian and Solaris osfamilies. Your osfamily is identified as <${::osfamily}>.")
     }
   }
+
+  if $ensure_vas == 'present' and ($::osfamily == 'RedHat' or $::osfamily == 'Debian' or ($::osfamily == 'Suse' and $::lsbmajdistrelease != '9')) {
```

Comment:
if ensure_vas is present on an unsupported platform, shouldnt it fail so that you know your configuration is not valid. This would seem to silently skip it.